scripts/pod_lib_lint.rb

```diff
@@ -83,6 +84,11 @@ def main(args)
       puts "Still working, running for #{number_of_times_checked / 60}min."
     end
   end
+
+  unless pod_lint_result == true
+    exit(1)
```

Comment:
This change is required to convert silent failure like [this](https://travis-ci.org/firebase/firebase-ios-sdk/jobs/562678041) to visible failure like [this](https://travis-ci.org/firebase/firebase-ios-sdk/jobs/562705029)